boards/arm/v2m_musca/doc/v2m_musca.rst

```diff
@@ -0,0 +1,412 @@
+- Cortex-M System Design Kit UART
+
+.. image:: img/v2m_musca.png
+     :width: 442px
+     :align: center
+     :height: 377px
+     :alt: ARM V2M Musca
+
+More information about the board can be found at the `V2M Musca Website`_.
+
+Hardware
+********
+
+ARM V2M MUSCA provides the following hardware components:
+
+
+
+- ARM Cortex-M33
+- ARM IoT Subsystem for Cortex-M33
+- Memory
+
+  - 128KB SRAM
+  - 2MB of external SRAM
+  - 8MB of external QSPI flash.
+
+- Debug
+
+  - JTAG, SWD & 4 bit TRACE
+  - DAPLink with a virtual UART port
+
+- Arduino interface
+
+  - 16 3V3 GPIO.
+  - UART.
+  - SPI.
+  - I2C.
+  - I2S.
+  - 3-channel PWM.
+  - 6-channel analog interface.
+
+- On-board Peripherals
+
+  - User RGB LED
+  - Gyro sensor
+  - Combined ADC/DAC/temperature sensor
+
+
+User push buttons
+=================
+
+The v2m_musca board provides the following user push buttons:
+• PBON power on/off.
+• nSRST: Cortex-M33 system reset and CoreSight debug reset.
+• ISP: Updates DAPLink firmware.
+• HWRST: Resets DAPLink.
+
+
+Supported Features
+===================
+
+The v2m_musca board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| TIMER     | on-chip    | timer                               |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+See the `V2M Musca Website`_ for a complete list of V2M Musca board hardware
+features.
+
+The default configuration can be found in the defconfig file:
+
+.. code-block:: console
+
+   boards/arm/v2m_musca/v2m_musca_defconfig
+
+Interrupt Controller
+====================
+
+Musca is a Cortex-M33 based SoC and has 15 fixed exceptions and 77 IRQs.
+
+A Cortex-M33-based board uses vectored exceptions. This means each exception
+calls a handler directly from the vector table.
+
+Handlers are provided for exceptions 1-7, 11-12, and 14-15. The table here
```

Comment:
Who provides these handlers?  Should this read:

    Zephyr provides handlers for exceptions 1-7, 11, 12, 14, and 15, as 
    listed in this table:

Reply:
Zephyr handles these exceptions for all Cortex M boards, why are we calling this out specifically on this board?

Reply:
@MaureenHelm This was added for consistency purposes, since it was included in the Beetle board documentation.It is also good to list the Secure-fault exception which is new to the v8m processors.